src/sentry/monitors/tasks.py

```diff
@@ -25,16 +26,16 @@
 #
 # NOTE: We should keep an eye on this as we have more and more usage of
 # monitors the larger the number of checkins to check will exist.
-MONITOR_LIMIT = 10_000
+MONITOR_LIMIT = 20_000
 
 # This is the MAXIMUM number of pending MONITOR CHECKINS this job will check.
 #
 # NOTE: We should keep an eye on this as we have more and more usage of
 # monitors the larger the number of checkins to check will exist.
-CHECKINS_LIMIT = 10_000
+CHECKINS_LIMIT = 20_000
```

Comment:
May want to consider only bumping this number since it seems much less expensive to run through the list of possible timed out check-ins versus going through 10,000 missed monitors and modifying their status.